docs/helm/eks-deployment.md

```diff
@@ -160,6 +160,8 @@ Now we have an EKS cluster up and running there are a few one time steps we need
         name: nfs-client
         parameters:
           directoryPerms: "700"
+          uid: "0"
+          gid: "0"
           fileSystemId: fs-SOMEUUID
           provisioningMode: efs-ap
         reclaimPolicy: Delete
```

Comment:
we need to state somewhere in this section that those parameters are only supported after a specific aws-efs-csi driver